packages/react-native-worklets/plugin-oxc/src/closure.rs

```
@@ -0,0 +1,228 @@
+        });
+    }
+
+}
+
+/// Drives visitor walks over a function's body + params, regardless of whether
+/// the function is a `Function` or `ArrowFunctionExpression`.
+pub trait WalkFunctionBody<'a> {
+    fn walk_into<V: Visit<'a>>(self, visitor: &mut V);
+}
+
+impl<'a, 'b> WalkFunctionBody<'a> for &'b Function<'a> {
```

Clippy (1.97.1)

[warning] the following explicit lifetimes could be elided

(warning)